database-mysql/index.js

```diff
@@ -25,6 +25,10 @@ const Messages = connection.define('messages', {
   toId: Sequelize.INTEGER
 })
 
+const getUsers = () => {
```

Comment:
please put db helpers in a separate file database-mysql/db-helpers.js (don't forget to require models)